components/account/AccountDisplayName.vue

```diff
@@ -3,6 +3,7 @@ import type { mastodon } from 'masto'
 
 defineProps<{
   account: mastodon.v1.Account
+  useShowEmojisPreference?: boolean
```

Comment:
Even if it will involve getting the preference in more places, I think it is better to change this prop to `hideEmojis` defaulting to false and opt-in where we want to remove them to be more explicit.